src/Decompiler/Decompiler.cs

```diff
@@ -269,20 +270,24 @@ public void Assemble(string fileName, Assembler asm)
         /// <param name="fileName"></param>
         /// <param name="arch"></param>
         /// <param name="platform"></param>
-        public Program LoadRawImage(string fileName, LoadDetails raw)
+        public Program LoadRawImage(string fileName, string outputDir, LoadDetails raw)
         {
             eventListener.ShowStatus("Loading raw bytes.");
             byte[] image = loader.LoadImageBytes(fileName, 0);
             var program = loader.LoadRawImage(fileName, image, null, raw);
-            Project = CreateDefaultProject(fileName, program);
+            Project = CreateDefaultProject(fileName, outputDir, program);
             eventListener.ShowStatus("Raw bytes loaded.");
             return program;
         }
 
-        protected Project CreateDefaultProject(string fileName, Program program)
+        protected Project CreateDefaultProject(string fileNameWithPath, string outputDir, Program program)
         {
-            program.Filename = fileName;
-            program.EnsureFilenames(fileName);
+            program.Filename = fileNameWithPath;
+
+            FileInfo fif = new FileInfo(fileNameWithPath);
+            string fileName = fif.Name;
+
+            program.EnsureFilenames(outputDir + "\\" + fileName);
```

Comment:
Platform-specific code

Reply:
I suggest you use `Path.Combine` here to avoid the platform specific code.